docs/CONFIG-VARS.md

```diff
@@ -231,15 +232,16 @@ Ubuntu 20.04 LTS is the operating system used on the Jump/NFS servers. Ubuntu cr
 
 Additional node pools can be created separate from the default node pool. This is done with the `node_pools` variable, which is a map of objects. Irrespective of the default values, the following variables are required for each node pool:
 
-| Name | Description | Type | Notes |
-| :--- | ---: | ---: | ---: |
-| machine_type | Type of the node pool VMs | string | |
-| os_disk_size | Disk size for node pool VMs in GB | number | |
-| min_nodes | Minimum number of nodes for the node pool | number | Value must be between 0 and 100. Setting min and max node counts to the same value disables autoscaling |
-| max_nodes | Maximum number of nodes for the node pool | number | Value must be between 0 and 100. Setting min and max node counts to the same value disables autoscaling |
-| max_pods | Maximum number of pods per node | number | Default is 110
-| node_taints | Taints for the node pool VMs | list of strings | |
-| node_labels | Labels to add to the node pool VMs | map | |
+| Name             |                               Description |            Type |                                                                                                   Notes |
+|:-----------------|------------------------------------------:|----------------:|--------------------------------------------------------------------------------------------------------:|
```

Comment:
Same comment regarding extra white space and column format line as above. Also makes it harder to see the one new value.